packages/main/test/samples/MultiInput.sample.html

```diff
@@ -0,0 +1,171 @@
+<style>
+	.fiori--dark .inputIcon,
+	.hcb .inputIcon {
+		color: #fafafa;
+		background: transparent;
+	}
+	.hcb .inputIcon:hover{
+		background: #7a5100;
+	}
+
+	.inputIcon {
+		background: #fafafa;
+		color: #0a6ed1;
+		cursor: pointer;
+	}
+	.inputIcon:hover{
+		background: rgba(224,224,224,0.5)
+	}
+</style>
+
+<header>
+	<h2 class="control-header">Multi Input</h2>
```

Comment:
Add the @ since marker and update the new components array in the playground project to appear as "new"